### PR TITLE
Save backup on Disk (#8272)

### PR DESCRIPTION
**Pull request by NicoMaio**
Implement saving the configuration backup on the Disk alongside the Redis backup.

### CODE REVIEW

File: scripts/lua/modules/system_config/backup_config.lua [hunk 1]
```diff
@@ -51,13 +58,113 @@ function backup_config.save_backup()
...
+
+    ntop.mkdir(base_dir)
+    local backup_files = {}
+
+    local last_b_name = "" -- backup more recent
+    local first_b_name = "" -- backup older
+
+    local num_backups = 0
+
+    for dir in io.popen(ls_files_command..[[ ]]..base_dir):lines() do
```
Comment (lucaderi) [inline scripts/lua/modules/system_config/backup_config.lua]: THis does not work on Windows

File: scripts/lua/modules/system_config/backup_config.lua [hunk 2]
```diff
@@ -51,13 +58,113 @@ function backup_config.save_backup()
...
+-- @brief Save configurations backup on File System.
+function backup_config.fs_save_backup(backup_time_key, backup, to_ignore)
+    
+    if debugger then
+        traceError(TRACE_DEBUG, TRACE_CONSOLE, "START BACKUP SAVING ON FS")
+    end
+
+    local key = backup_time_key
+
+    local ls_files_command      = ls_file_command
```
Comment (lucaderi) [inline scripts/lua/modules/system_config/backup_config.lua]: Please use ntop.readdir()

**Review by lucaderi — COMMENTED** [review]
Please fix the problems reported

File: scripts/lua/modules/system_config/backup_config.lua (hunk 1 above)
Comment (NicoMaio) [reply scripts/lua/modules/system_config/backup_config.lua]: Corrected by commit d535f22fb01e6332ebbba6cee1ca0ae9226e0316.

File: scripts/lua/modules/system_config/backup_config.lua (hunk 2 above)
Comment (NicoMaio) [reply scripts/lua/modules/system_config/backup_config.lua]: Corrected by commit d535f22fb01e6332ebbba6cee1ca0ae9226e0316.

**Pull request merged**